packages/lodestar/src/api/impl/node/node.ts

```diff
@@ -43,6 +46,7 @@ export class NodeApi implements INodeApi {
   public async getPeers(): Promise<NodePeer[]> {
     return this.network.getPeers().map((peer) => {
       const conn = this.network.getPeerConnection(peer);
+      if (!conn) throw Error(`No peer connection for ${peer.id.toString()}`);
```

Comment:
Should it throw an error here? Or maybe ignore this peer

Reply:
We should ignore.  Not sure but I think network.getPeers doesn't return peers without connection anyways